src/+otp/+sanzserna/SanzSernaProblem.m

```diff
@@ -4,6 +4,11 @@
     % applied to evolutionary problems in partial differential equations. Numerische Mathematik, 50(4), 405–418.
     % https://doi.org/10.1007/BF01396661
     
+    properties (SetAccess = private)
+        RhsLinear
+        RhsForcing
```

Comment:
Wouldn't it be better to somehow group the various splittings into one? I know we had this discussion before, and we did not come to a good conclusion. I personally would prefer something like

RhsLinearForcing, where RhsLinearForcing{1} is RhsLinear, and RhsLinearForcing{2} is RhsForcing, but...